localstack/testing/pytest/util.py

```diff
@@ -0,0 +1,24 @@
+import os
+import pwd
+from multiprocessing import Process, ProcessError
+from typing import Callable
+
+
+def run_as_os_user(target: Callable, uid: str | int, gid: str | int = None):
```

Comment:
Note: currently only accepting a `Callable` here - could be extended with `*args, **kwargs` in the future, as needed. 👍